src/TableView/TableView.jsx

```diff
@@ -261,9 +338,11 @@ class TableView extends Component {
           handlePageChange={this.handlePageChange} createData={this.props.createData}
           removeData={this.handleDeleteData} updateData={this.props.updateData}
           filterData={this.handleFilterData} visibleColumns={headers}
-          sortData={this.handleSortData} openModal={this.handleOpenModal}
-          closeModal={this.handleCloseModal} editData={this.handleEditItem}
-          openAlert={this.handleOpenAlert}
+          openModal={this.handleOpenModal} closeModal={this.handleCloseModal}
```

Comment:
It wasn't intended to put one attribute per line (merging ;) ). But I don't think that putting one-per-line is good - the code will be sooooo loooong.